src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action in actions) {
+    if (actions[action]['type'] === 'addProperties') {
+      Object.assign(state, actions[action].extraData);
+    }
+
+    if (actions[action]['type'] === 'removeProperties') {
+      for (const del of actions[action].keysToRemove) {
+        delete state[del];
+      }
+    }
+
+    if (actions[action]['type'] === 'clear') {
```

Comment:
Access the 'type' property using dot notation (actions[action].type) instead of bracket notation